src/security/credentials.rs

```diff
@@ -125,6 +190,57 @@ impl ServerCredentialsBuilder {
         self
     }
 
+    /// Finalize the [`ServerCredentialsBuilder`] with a user-defined fetcher
+    /// and build the [`ServerCredentials`].
+    pub fn build_with_fetcher(
```

Comment:
It's unnecessary to build fetcher using a builder. You can move it to server::Binder.

Reply:
Fixed